runtime/doc/tabpage.txt

```diff
@@ -110,12 +119,18 @@ something else.
 		- When 'hidden' is not set, [!] is not used, a buffer has
 		  changes, and there is no other window on this buffer.
 		Changes to the buffer are not written and won't get lost, so
-		this is a "safe" command.
+		this is a "safe" command. >
+                    :tabclose   " close the current tab page
 
+:{count}tabc[lose][!]
```

Comment:
What's the syntactic purpose of using braces here as opposed to square brackets? It seems like they're being arbitrarily interchanged (for example, `:[range]argdo[!]` above, where it seems like the brackets mean an optional part of the command.